.gitignore

```diff
@@ -2,6 +2,8 @@
 .vscode/
 *.iml
 .env
+.sqlite
```

Comment:
The entry `.sqlite` seems unusual. Typically, you would ignore specific SQLite database files like `*.sqlite` or `db.sqlite3`. Consider revising this entry if it was not intended.